From 181634d94e8101d80a167db6101613ba3d18d701 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 6 Nov 2018 15:03:17 +0100
Subject: [PATCH 2/2] [SQUSH][DNM] Merge tag 'android-9.0.0_r16' into
 lineage-16.0

commit fbfde0bae50b9a6b01f56c80323ca22a6c4e14f0
Author: Jack Yu <jackyu@google.com>
Date:   Tue Jun 26 14:58:55 2018 -0700

    Fixed ETWS channel settings for KDDI

    Changed the ETWS channel settings based on the new
    KDDI open device requirement.

    Test: Manual
    Bug: 110233310
    Change-Id: Iedadb4d09245029e7146a96c4921fb1bc56867e1

commit d22a15fa4a33811fbe8a8a556744e2ce04abe712
Author: Bill Yi <byi@google.com>
Date:   Wed Jun 20 17:10:46 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: I5df1b75b75267f1d8af4820ac48fc10f6c6da5d4

Change-Id: Ie8d07657c1f409df81801f46e7381df0160a5d0f
---
 res/values-mcc440-mnc50/config.xml | 9 ++++++---
 res/values-mcc440-mnc51/config.xml | 9 ++++++---
 res/values-mcc440-mnc52/config.xml | 9 ++++++---
 res/values-uz/strings.xml          | 2 +-
 4 files changed, 19 insertions(+), 10 deletions(-)

diff --git a/res/values-mcc440-mnc50/config.xml b/res/values-mcc440-mnc50/config.xml
index 0db8f82..7852691 100644
--- a/res/values-mcc440-mnc50/config.xml
+++ b/res/values-mcc440-mnc50/config.xml
@@ -15,8 +15,11 @@
 -->
 
 <resources>
-    <string-array name="additional_cbs_channels_strings" translatable="false">
-        <item>0xAC00:type=test, emergency=true, scope=carrier</item>
-        <item>0x1105-0xAFFF:type=other, emergency=true, scope=carrier</item>
+    <!-- Based on KDDI open device requirement, only 4352 and 40963 should be enabled -->
+    <string-array name="etws_alerts_range_strings" translatable="false">
+        <!-- 0x1100 for earthquake -->
+        <item>0x1100:rat=gsm, emergency=true</item>
+        <!-- 0xA003 for Tsunami -->
+        <item>0xA003:rat=gsm, emergency=true, scope=carrier</item>
     </string-array>
 </resources>
diff --git a/res/values-mcc440-mnc51/config.xml b/res/values-mcc440-mnc51/config.xml
index 0db8f82..7852691 100644
--- a/res/values-mcc440-mnc51/config.xml
+++ b/res/values-mcc440-mnc51/config.xml
@@ -15,8 +15,11 @@
 -->
 
 <resources>
-    <string-array name="additional_cbs_channels_strings" translatable="false">
-        <item>0xAC00:type=test, emergency=true, scope=carrier</item>
-        <item>0x1105-0xAFFF:type=other, emergency=true, scope=carrier</item>
+    <!-- Based on KDDI open device requirement, only 4352 and 40963 should be enabled -->
+    <string-array name="etws_alerts_range_strings" translatable="false">
+        <!-- 0x1100 for earthquake -->
+        <item>0x1100:rat=gsm, emergency=true</item>
+        <!-- 0xA003 for Tsunami -->
+        <item>0xA003:rat=gsm, emergency=true, scope=carrier</item>
     </string-array>
 </resources>
diff --git a/res/values-mcc440-mnc52/config.xml b/res/values-mcc440-mnc52/config.xml
index 0db8f82..7852691 100644
--- a/res/values-mcc440-mnc52/config.xml
+++ b/res/values-mcc440-mnc52/config.xml
@@ -15,8 +15,11 @@
 -->
 
 <resources>
-    <string-array name="additional_cbs_channels_strings" translatable="false">
-        <item>0xAC00:type=test, emergency=true, scope=carrier</item>
-        <item>0x1105-0xAFFF:type=other, emergency=true, scope=carrier</item>
+    <!-- Based on KDDI open device requirement, only 4352 and 40963 should be enabled -->
+    <string-array name="etws_alerts_range_strings" translatable="false">
+        <!-- 0x1100 for earthquake -->
+        <item>0x1100:rat=gsm, emergency=true</item>
+        <!-- 0xA003 for Tsunami -->
+        <item>0xA003:rat=gsm, emergency=true, scope=carrier</item>
     </string-array>
 </resources>
diff --git a/res/values-uz/strings.xml b/res/values-uz/strings.xml
index e61be35..d35aca0 100644
--- a/res/values-uz/strings.xml
+++ b/res/values-uz/strings.xml
@@ -23,7 +23,7 @@
     <string name="no_cell_broadcasts" msgid="5409324337492758562">"Ayni paytda siz joylashgan hudud uchun hech qanday faol ogohlantirish yo‘q. Ogohlantirish parametrlarini Sozlamalar menyusidan o‘zgartirish mumkin."</string>
     <string name="menu_delete_all" msgid="3940997343921149800">"Tarqatma xabarlarni o‘chirish"</string>
     <string name="message_options" msgid="3178489901903589574">"Xabar sozlamalari"</string>
-    <string name="menu_view_details" msgid="1040989019045280975">"Tafsilotlarni ko‘rish"</string>
+    <string name="menu_view_details" msgid="1040989019045280975">"Tafsilotlar"</string>
     <string name="menu_delete" msgid="128380070910799366">"Tarqatma xabarni o‘chirish"</string>
     <string name="view_details_title" msgid="1780427629491781473">"Ogohlantirish tafsilotlari"</string>
     <string name="confirm_delete_broadcast" msgid="2540199303730232322">"Ushbu tarqatma xabar o‘chirib tashlansinmi?"</string>
-- 
2.17.1

